.rubocop.yml

```diff
@@ -4,12 +4,12 @@
 # - submit the change in the code_generator
 
 AllCops:
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.1
```

Comment:
Same comment as for cookstyle, I would prefer 3.0.3 here.

Reply:
Done